index.html

```diff
@@ -25,7 +25,7 @@ <h1>Hello, Welcome to my Portfolio!</h1>
       <h1 class="project-tile">My Projects</h1>
```

Comment:
1. Please kindly add alt to the code to provide alternative information for the user for some reason cannot view it.

Reply:
@hagan199, you have given a reason that is not concise for using `alt`. `alt` is used for images, so that visually impaired users can know what is in the position of that image as well as for screen readers.